e2e/kube/storage/examples_storage_test.go

```diff
@@ -34,6 +47,36 @@ var _ = Describe("Examples", func() {
 				err = kubectlHelper.WaitForPVC(namespace, "nats-deployment-nats-pvc")
 				Expect(err).ToNot(HaveOccurred())
 			})
+
+			It("extended-statefulset pvc example must work", func() {
+				yamlFilePath := examplesDir + "extended-statefulset/exstatefulset_pvcs.yaml"
+
+				// Adding storageclass to volumeclaimtemplates
+				class, ok := os.LookupEnv("OPERATOR_TEST_STORAGE_CLASS")
+				Expect(ok).To(Equal(true))
+
+				err := kubectlHelper.AddTestStorageClassToVolumeClaimTemplates(yamlFilePath, class)
```

Comment:
If I read the code correctly this actually alters the file in the `exampes` directory, depending on the environment variable. I think this file shouldn't be changed.

Reply:
Done.